components/timestamp/timestamp.tsx

```diff
@@ -278,7 +284,8 @@ class Timestamp extends PureComponent<Props, State> {
             useRelative = (): ResolvedFormats['relative'] => {
                 const {
                     display,
-                    updateIntervalInSeconds,
+                    updateIntervalInSeconds = this.props.updateIntervalInSeconds,
```

Comment:
I'm not familiar with the `autoRange` pattern here, but can you elaborate on why this line is changing?

Reply:
(`autoRange` just determines which `RangeDescriptor` to use.)

This line specifically provides a default `updateIntervalInSeconds` from props if the range descriptor doesn't provide one itself.

(The fallback to both of these are statically defined intervals based on the display unit.)